credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt

```diff
@@ -316,23 +320,64 @@ class Web5TestVectorsCredentials {
 
   @Test
   fun create() {
-    val typeRef = object : TypeReference<TestVectors<CreateTestInput, String>>() {}
+    val typeRef = object : TypeReference<TestVectors<CreateTestInput, Map<String, Any>?>>() {}
     val testVectors = mapper.readValue(File("../web5-spec/test-vectors/credentials/create.json"), typeRef)
 
     testVectors.vectors.filter { it.errors == false }.forEach { vector ->
-      val vc = VerifiableCredential.fromJson(mapper.writeValueAsString(vector.input.credential))
-      val portableDid = Json.parse<PortableDid>(Json.stringify(vector.input.signerPortableDid!!))
+      var issuanceDate = Date()
+      vector.input.credential?.get("issuanceDate")?.let {
+        issuanceDate = Date.from(Instant.parse(it as String))
+      }
 
-      val keyManager = InMemoryKeyManager()
-      val bearerDid = BearerDid.import(portableDid, keyManager)
-      val vcJwt = vc.sign(bearerDid)
+      var expirationDate: Date? = null
+      vector.input.credential?.get("expirationDate")?.let {
+        expirationDate = Date.from(Instant.parse(it as String))
+      }
 
-      assertEquals(vector.output, vcJwt, vector.description)
-    }
+      var credentialStatus: CredentialStatus? = null
+      (vector.input.credential?.get("credentialStatus") as? Map<String, Any>)?.let {
+        credentialStatus = CredentialStatus.fromMap(it)
+      }
 
-    testVectors.vectors.filter { it.errors == true }.forEach { vector ->
-      assertFails(vector.description) {
-        VerifiableCredential.fromJson(mapper.writeValueAsString(vector.input.credential))
+      val evidence = vector.input.credential?.get("evidence") as? List<Any>
+
+      val vc = VerifiableCredential.create(
+        type = vector.input.credential?.get("type") as String,
```

Comment:
there's a lot of `?get("...")` here - have you considered having concrete types for these fields inside vector or credential input?